src/serac/physics/solid.hpp

```diff
@@ -270,6 +270,14 @@ class Solid : public BasePhysics {
   void setMaterialParameters(std::unique_ptr<mfem::Coefficient>&& mu, std::unique_ptr<mfem::Coefficient>&& K,
                              bool material_nonlin = true);
 
+  /**
+   * @brief Set the adjoint load
+   * @note The ParLinearForm must be built on the same finite element space as the displacement
+   *
+   * @param adjoint_load The adjoint load to apply
+   */
+  void setAdjointLoad(mfem::ParLinearForm& adjoint_load);
```

Comment:
This was the result of talking with @goxberry and the LiDO team. I believe the main point is they want to be able to differentiate between primal and dual variables.

Reply:
ping @sethwatts @pmm14006

Reply:
Yes, this is an area we're trying to be fastidious about in current LiDO. Also, from a practical standpoint, the adjoint load is going to come from integrating derivatives over the domain and/or boundaries, so it's already in a linear form format rather than a GridFunction.